plugins/org.jkiss.dbeaver.ext.import_config/src/org/jkiss/dbeaver/ext/import_config/wizards/sqlworkbench/ConfigImportWizardPageSqlWorkbenchConnections.java

```diff
@@ -0,0 +1,213 @@
+
+        mysqlDriver = new ImportDriverInfo(
+            mysql8.getId(),
+            mysql8.getName(),
+            mysql8.getSampleURL(),
+            mysql8.getDriverClassName());
+    }
+
+    @Override
+    protected void loadConnections(@NotNull ImportData importData) throws DBException {
+        setErrorMessage(null);
+
+        ConfigImportWizardSqlWorkbench wizard = (ConfigImportWizardSqlWorkbench) getWizard();
+        File inputFile = wizard.getInputFile();
+        try (InputStream is = new FileInputStream(inputFile)) {
+            try (Reader reader = new InputStreamReader(is)) {
+                importXML(importData, reader);
+            }
+        } catch (Exception e) {
+            log.warn("Exception during to load connections", e);
+            setErrorMessage(e.getMessage());
+        }
+    }
+
+    private void importXML(ImportData importData, Reader reader) throws XMLException {
+        Document document = XMLUtils.parseDocument(reader);
+        Collection<Element> allElements = XMLUtils.getChildElementList(XMLUtils.getChildElementList(document.getDocumentElement())
+            .iterator().next());
+        for (Element conElement : allElements) {
+            importData.addConnection(parseConnection(conElement));
+        }
+    }
+
+    private ImportConnectionInfo parseConnection(Element conElement) {
+
+        String alias = getElementValueOrEmptyString(conElement, "name");
+        Element parameterValues = getElemByNameAndTagKeyValue(conElement, "parameterValues");
+        if (parameterValues == null) {
+            throw new ImportConfigurationException("Can't find parameterValues tag in the xml");
+        }
+        String hostName = getElementValueOrEmptyString(parameterValues, "hostName");
+        String port = getElementValueOrEmptyString(parameterValues, "port");
+        String userName = getElementValueOrEmptyString(parameterValues, "userName");
+
+
+        ImportConnectionInfo connectionInfo = new ImportConnectionInfo(
+            mysqlDriver,
+            null,
+            alias,
+            null,
+            hostName,
+            port,
+            null,
+            userName,
+            null);
+
+        configureSsh(parameterValues, connectionInfo);
+        configureSsl(parameterValues, connectionInfo);
+        configureDriverProperties(parameterValues, connectionInfo);
+        return connectionInfo;
+    }
+
+    private void configureDriverProperties(Element parameterValues, ImportConnectionInfo connectionInfo) {
+        for (Element element : XMLUtils.getChildElementList(parameterValues)) {
+            if (!keyWords.contains(element.getAttribute("key"))) {
+                String driverPropertyValue = XMLUtils.getElementBody(element);
+                connectionInfo.setProperty(element.getAttribute("key"), driverPropertyValue);
+            }
+        }
+    }
+
+    private @Nullable String getElementValueOrEmptyString(@Nullable Element conElement, @NotNull String keyValue) {
```

Comment:
We usually write it at the top
```suggestion
    @Nullable
    private String getElementValueOrEmptyString(@Nullable Element conElement, @NotNull String keyValue) {
```